config/pod.cache2disk.yaml.tim

```diff
@@ -61,6 +61,8 @@ spec:
     env:
     - name: SD_PUSHGATEWAY_URL
       value: "{{pushgateway_url}}"
+    - name: SD_TERMINATION_GRACE_PERIOD_SECONDS
+      value: {{termination_grace_period_seconds}}
```

Comment:
using this to control wait in run.sh as a workaround to wait for launcher to execute abort steps. as discussed with @pritamstyz4ever we will revisit this